components/Status.tsx

```diff
@@ -81,3 +81,23 @@ export default function Status({ statusData, coordinatorName }) {
     </li>
   );
 }
+
+
+function scriptTypeToString(scriptType: string): string {
+  switch (scriptType.toLowerCase()) {
+    case 'witness_v0_keyhash':
+      return 'P2WPKH';
+    case 'witness_v0_scripthash':
+      return 'P2WSH';
+    case 'witness_v1_taproot':
+      return 'Taproot';
+    case 'scripthash':
+      return 'P2SH';
+    case 'pubkeyhash':
+      return 'P2PKH';
+    case 'pubkey':
+      return 'P2PK';
```

Comment:
Do you think it'd be good to put an example prefix for the address too?

Ie "P2WPKH (bc1q...)"

Reply:
Yeah I think that would be a helpful addition

Reply:
Added. scary that copilot autofilled some of these for me